src/getting_started/linux_x86.md

```diff
@@ -0,0 +1,107 @@
+# Quickstart for Linux on x86
+
+Run Parsec with the Mbed Crypto provider natively on your machine.
+
+## Overview
+
+In this guide you will learn how to quickly get started with Parsec in a toy environment. More
+specifically you will:
+
+- download and start the Parsec service
+- perform some Parsec commands using `parsec-tool`
+- see (and maybe run) Rust and Go clients code example
+
+Note that this installation method is only made for educational purposes and is not secure. To
+securely install Parsec on Linux for production, check [this guide
+instead](../parsec_service/install_parsec_linux.md).
+
+## Prerequisites
+
+- a 64 bit Linux running on x86
+
+The commands should be run as a normal user (not root).
+
+## Downloading Parsec
+
+Run the following command to download and unpack the `linux_x86` folder. It contains the Parsec
+service (`parsec`), its pre-built configuration (`config.toml`) and the Parsec tool (`parsec-tool`).
+
+```
+curl -s -N -L https://github.com/parallaxsecond/parsec/releases/download/0.6.0/linux_x86.tar.gz | tar xz
```

Comment:
I think the folder inside the tarball should be called `linux_x86`, but I wouldn't call the file that since it will be tied to the version number. It might give the right impression to someone who looks at the release stuff alone, not related to this guide.

Reply:
Will rename both with the `parsec-0.7.0-linux_x86` name as SPIFFE does so it's clearer.

Reply:
No, what I meant was that just saying something generic like that is not a good idea because someone might think that's the whole of `parsec`, packaged for Linux x86 systems (and it's not). The thing added to the release, at least, should be named more specifically, though a good name doesn't come to mind right now

Reply:
ah I see your point. I can name it `quickstart-0.7.0-linux_x86`?

Reply:
Yes, I think that's good! maybe you can also throw in a `README` file that says what the package is for?